Functions.Templates/Templates/TimerTrigger-Python-Preview/timer_trigger_template.md

```diff
@@ -0,0 +1,47 @@
+# Azure Functions: Timer Trigger in Python
+
+## Timer Trigger
+
+A timer trigger lets you run a function on a schedule.
+
+## Using the Template
+
+Following is an example code snippet for Timer Trigger using the [new programming model](https://aka.ms/pythonprogrammingmodel) (currently in Preview).
+
+```python
+import datetime
+
+import logging
+
+import azure.functions as func
+
+app = func.FunctionApp()
+
+@app.function_name(name="mytimer")
+@app.schedule(schedule="0 */5 * * * *", arg_name="mytimer", run_on_startup=True,
+              use_monitor=False) 
```

Comment:
```suggestion
@app.schedule(schedule="0 */5 * * * *", arg_name="mytimer") 
```

Simplifying this - Please remove function_nameand other optional fields.

Reply:
@shreyabatra4 any thoughts on this?

Reply:
I feel like it's helpful to have function_name as well as other optional fields to demonstrate that they exist and can be used. Maybe I can add a comment stating they are optional to bring that clarity?